examples/01_sparql_queries.ipynb

```diff
@@ -244,435 +162,353 @@
+    "q"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Logical operators\n",
+    "What happened here? The function tried to find the shortest path between the `Pizza` and `PeperoniSausageTopping` classes. Since both are subclasses of `Food`, it initially chose the `hasIngredient` property.\n",
     "\n",
-    "Logical operators currently supported are & and |. These operators, when applied, aggregates the condition between two terms|"
+    "However, by increasing the `num_paths` parameter, we can get alternative paths. In this case, the second option uses `hasTopping`, which is the more appropiate query."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
+   "execution_count": 38,
    "metadata": {},
    "outputs": [
     {
-     "data": {
-      "text/plain": [
-       "cmso:hasChemicalSymbol\n",
-       "A data property linking an element with its chemical symbol."
-      ]
-     },
-     "execution_count": 17,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "(onto.terms.cmso.hasChemicalSymbol=='Al') & (onto.terms.cmso.hasElementRatio==1.0)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 18,
-   "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "cmso:hasChemicalSymbol\n",
-       "A data property linking an element with its chemical symbol."
-      ]
-     },
-     "execution_count": 18,
-     "metadata": {},
-     "output_type": "execute_result"
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
+      "PREFIX pizza: <https://example.org/pizza#>\n",
+      "SELECT DISTINCT ?Pizza ?PeperoniSausageTopping\n",
+      "WHERE {\n",
+      "    ?Pizza pizza:hasIngredient ?PeperoniSausageTopping .\n",
+      "    ?Pizza rdf:type pizza:Pizza .\n",
+      "    ?PeperoniSausageTopping rdf:type pizza:PeperoniSausageTopping .\n",
+      "} PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>\n",
+      "PREFIX pizza: <https://example.org/pizza#>\n",
+      "SELECT DISTINCT ?Pizza ?PeperoniSausageTopping\n",
+      "WHERE {\n",
+      "    ?Pizza pizza:hasTopping ?PeperoniSausageTopping .\n",
+      "    ?Pizza rdf:type pizza:Pizza .\n",
+      "    ?PeperoniSausageTopping rdf:type pizza:PeperoniSausageTopping .\n",
+      "}\n"
+     ]
     }
    ],
    "source": [
-    "(onto.terms.cmso.hasChemicalSymbol=='Al') | (onto.terms.cmso.hasElementRatio==1.0)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## @ operator\n",
-    "\n",
-    "The final class of operator we have is the @ operator. This can be used for resolving terms that has multiple paths. For example, rdfs:label which multiple entities can have. "
+    "q = onto.create_query(onto.terms.pizza.Pizza, onto.terms.pizza.PeperoniSausageTopping,num_paths=2)\n",
```

Comment:
I find it slightly confusing that the output of the cell 38, i.e. this one:

```python
q = onto.create_query(onto.terms.pizza.Pizza, onto.terms.pizza.PeperoniSausageTopping,num_paths=2)
print (q[0], q[1])
```

does not correspond to the one from cell 37. I think easier to understand would be if they are both pandas data frame, or if there's another cell between 38 and 39 to show the pandas data frame. In any case this way the transition to cell 39 would become also smoother.